packages/framework/src/client/__tests__/scheduler.spec.js

```diff
@@ -489,66 +694,218 @@ describe( 'Scheduler', () => {
 		} );
 
 		it( 'clears out completed and failed requests', () => {
-			const scheduler = new Scheduler( () => {}, () => {}, () => {} );
+			const operations = {
+				read: jest.fn(),
+			};
+			const scheduler = new Scheduler( operations, () => {}, () => {} );
 
-			scheduler.scheduleRequest( 'scheduledResource', {}, {}, now );
+			scheduler.scheduleRequest( {}, {}, 'scheduledResource', 'read', undefined, now );
 			scheduler.requests[ 0 ].getStatus = () => STATUS.scheduled;
 
-			scheduler.scheduleRequest( 'completeResource', {}, {}, now );
+			scheduler.scheduleRequest( {}, {}, 'completeResource', 'read', undefined, now );
 			scheduler.requests[ 1 ].getStatus = () => STATUS.complete;
 
-			scheduler.scheduleRequest( 'failedResource', {}, {}, now );
+			scheduler.scheduleRequest( {}, {}, 'failedResource', 'read', undefined, now );
 			scheduler.requests[ 2 ].getStatus = () => STATUS.failed;
 
 			expect( scheduler.requests.length ).toBe( 3 );
 
-			scheduler.cleanUp( now );
+			scheduler.cleanUp();
 
 			expect( scheduler.requests.length ).toBe( 1 );
 			expect( scheduler.requests[ 0 ].getStatus() ).toBe( STATUS.scheduled );
 		} );
+
+		it( 'combines data from multiple requests into one operation call', () => {
```

Comment:
Was this meant to be under `describe( 'sendReadyRequests' )`?
Same with `calls dataRequested with resource names`, and `calls dataReceived with resource names and results`.

Reply:
Yes, good catch. Thanks! It must have gotten moved around in the rebase/merges.

Reply:
Fixed in 8ca82d44926d2e44786c0064deecb1ec248262f6